app/javascript/courses/review/components/CoursesReview__Editor.res

```diff
@@ -1069,6 +1132,19 @@ let make = (
     pending ? GradesEditor : ReviewedSubmissionEditor(OverlaySubmission.grades(overlaySubmission))
   }
 
+  React.useEffect0(() => {
+    switch submissionReport {
+    | Some(report) => {
+        let intervalId = Js.Global.setInterval(
+          () => reloadSubmissionReport(report, updateSubmissionReportCB),
+          30000,
+        )
```

Comment:
The attempt to reload the submission report should only be done if tests are queued or in progress. With the current implementation, these queries will be repeated indefinitely even after automated tests have completed.

Reply:
@harigopal If you check the implementation of `reloadSubmissionReport`. I guess that's already taken care of? 

```rescript
let reloadSubmissionReport = (report, updateSubmissionReportCB) => {
  switch SubmissionReport.status(report) {
  | Pending => loadSubmissionReport(report, updateSubmissionReportCB)
  | Success
  | Failure
  | Error => ()
  }
}